src/main/webapp/app/exercises/programming/shared/code-editor/ace/code-editor-ace.scss

```diff
@@ -76,4 +76,6 @@ Ace Editor
         transition: background-color 0s;
         background-color: var(--code-editor-gutter-newline-background);
     }
+
+    @import '../../../../../shared/markdown-editor/ace-editor/ace-editor';
```

Comment:
I'm not happy with imports like this. If the ``ace-editor`` component is used in this wrapper component, shouldn't it be possible to specify these styles directly in the ``ace-editor`` component rather than importing it into various locations? This looks pretty hacky.

Moreover, I think you imported the styles to the markdown and code editors, but the `ace-editor` component is used in a couple more locations which aren't checked at the moment and where the styles wouldn't apply, yet the search would be active there as well as it's imported into the `ace-editor`component.

What do you think?